### PR TITLE
EDS Deployment Samples 1.0.0

### PR DESCRIPTION
**Pull request by dcbroad3**
Adds EDS install sample and EDS Azure IOT deployment sample.

### CODE REVIEW

**Comment by derekendres** [comment]
My couple of comments.  I think there should be some indication in the readme of what you get out of the box with the EDS configuration files.  I think in azure you just get storage.  For install you get some dummy Modbus data selection and Modbus data source (possibly we should clean those files too...)

File: deployment_samples/EdgeDataStore/Install/HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Version History
+
+## 1.0.0 / 2020-05-13
```
Comment (dcbroad3) [inline deployment_samples/EdgeDataStore/Install/HISTORY.md]: Standard reminder to update this date when ready

File: deployment_samples/EdgeDataStore/Install/reset-device.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+echo "Reset Device: Uninstall IoT Edge runtime..."
```
Comment (dcbroad3) [inline deployment_samples/EdgeDataStore/Install/reset-device.sh]: Copy/pasta error

File: deployment_samples/EdgeDataStore/Install/templates/PIEgress.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+,{
+  "Id": "<egressID>",
+  "ExecutionPeriod": "00:00:10",
+  "NamespaceId": "<database>",
+  "StreamPrefix": "<prefix>",
+  "Endpoint": "<url>",
+  "UserName": "<id>",
+  "Password": "<password>",
+  "ValidateEndpointCertificate": false
+}
```
Comment (dcbroad3) [inline deployment_samples/EdgeDataStore/Install/templates/PIEgress.json:10]: I can't comment on the file as it's binary, but you included `/Install/templates/installation_files/EdgeDataStore_linux-x64.deb`, which definitely should not be checked in.

File: deployment_samples/EdgeDataStore/Install/templates/installation_files/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Add your installation files here.
```
Comment (dcbroad3) [inline deployment_samples/EdgeDataStore/Install/templates/installation_files/README.md]: I would probably say `Add your installation files to this folder` (or directory)

File: deployment_samples/EdgeDataStore/Install/templates/silent.ini [hunk 5]
```diff
@@ -0,0 +1,3 @@
+5591
```
Comment (dcbroad3) [inline deployment_samples/EdgeDataStore/Install/templates/silent.ini]: Shouldn't this just be 5590?

**Review by derekendres — APPROVED** [review]

**Pull request merged**